openassessment/xblock/openassessmentblock.py

```diff
@@ -615,6 +617,44 @@ def student_view(self, context=None):  # pylint: disable=unused-argument
         template = get_template("openassessmentblock/base.html")
         return self._create_fragment(template, context_dict, initialize_js_func='OpenAssessmentBlock')
 
+    @property
+    def mfe_views_supported(self):
+        """
+        Currently, there are some unsupported use-cases for ORA MFE views.
+
+        Unsupported use-cases:
+        1) Team assignments
+        2) Assignments with reordered assessment steps
+
+        Returns:
+        - False if we are in one of these unsupported configurations.
+        - True otherwise.
+        """
+
+        # Team assessments are currently unsupported
+        if self.is_team_assignment():
+            return False
+
+        # Determine if our steps have been reordered (omission of steps is fine)
+        default_assessment_order = copy.copy(self.VALID_ASSESSMENT_TYPES)
+        current_assessment_order = self.assessment_steps
+        is_default_order = True
+
+        last_step_index = 0
+        for assessment_step in current_assessment_order:
+            step_index = default_assessment_order.index(assessment_step)
+
+            if step_index < last_step_index:
+                is_default_order = False
+                break
+            last_step_index = step_index
+
+        # Assessment step reordering is currently unsupported
+        if not is_default_order:
+            return False
```

Comment:
Pull all this out into its own function, is_default_step_order. It will make mfe_views_supported more readable and it will avoid the "break and also check flag" thing, you'll just be able to return